git-cliff-core/src/changelog.rs

```diff
@@ -250,11 +253,9 @@ impl<'a> Changelog<'a> {
                 .map(|v| v.contains_variable(github::TEMPLATE_VARIABLES))
                 .unwrap_or(false)
         {
-            debug!(
-                "You are using an experimental feature! Please report bugs at <https://git-cliff.org/issues>"
-            );
+            log::warn!("GitHub integration is an experimental feature! Functionality may change or break. Please report bugs at <https://git-cliff.org/issues>");
```

Comment:
I changed this back from warn to debug a couple of months ago. Let's keep it as debug.